go/consensus/cometbft/light/service.go

```diff
@@ -179,13 +182,11 @@ func (c *client) worker() {
 	}
 	if err = trustLocalBlock(c.ctx, lastRetainedHeight); err != nil {
 		c.logger.Error("failed to store last retained block into trust store", "err", err)
-		return
 	}
 
 	// Store latest block into trust store.
 	if err = trustLocalBlock(c.ctx, consensus.HeightLatest); err != nil {
```

Comment:
Can this be removed, as `ch` will receive current epoch immediately?

Reply:
No, as the light client constructor would fail if there are no light blocks available in store. So in case both of these fail, the entire light client construction would fail (which is correct and it would be an unusual case, signifying that something must be wrong with the node).